step-maven-plugin/src/test/java/step/plugins/maven/ExecuteAutomationPackageMojoTest.java

```diff
@@ -19,159 +19,45 @@
 package step.plugins.maven;
 
 import org.apache.maven.artifact.Artifact;
-import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.project.MavenProject;
-import org.bson.types.ObjectId;
 import org.junit.Assert;
 import org.junit.Test;
-import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 import step.automation.packages.client.AutomationPackageClientException;
-import step.automation.packages.client.RemoteAutomationPackageClientImpl;
-import step.client.executions.RemoteExecutionFuture;
-import step.client.executions.RemoteExecutionManager;
-import step.core.artefacts.reports.ReportNodeStatus;
-import step.core.execution.model.AutomationPackageExecutionParameters;
-import step.core.execution.model.Execution;
-import step.core.execution.model.ExecutionMode;
-import step.core.execution.model.ExecutionStatus;
-import step.core.plans.filters.PlanByIncludedNamesFilter;
-import step.core.repositories.ImportResult;
-
-import java.io.File;
+import step.cli.AbstractExecuteAutomationPackageTool;
+
 import java.net.URISyntaxException;
-import java.util.*;
-import java.util.concurrent.TimeoutException;
-import java.util.stream.Collectors;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
 
 public class ExecuteAutomationPackageMojoTest extends AbstractMojoTest {
 
 	@Test
-	public void testExecuteOk() throws Exception {
-		Execution execution = getMockedExecution(ReportNodeStatus.PASSED);
-
-		List<Execution> executions = List.of(execution);
-		List<String> executionIds = getExecuteAutomationPackageResult(executions);
-
-		RemoteExecutionManager remoteExecutionManagerMock = createExecutionManagerMock(executions);
+	public void testExecute() throws Exception {
+		ExecuteAutomationPackageMojoTestable mojo = new ExecuteAutomationPackageMojoTestable();
 
-		RemoteAutomationPackageClientImpl remoteAutomationPackageClientMock = Mockito.mock(RemoteAutomationPackageClientImpl.class);
-		Mockito.when(remoteAutomationPackageClientMock.executeAutomationPackage(Mockito.any(), Mockito.any())).thenReturn(executionIds);
-
-		RunAutomationPackageMojoTestable mojo = new RunAutomationPackageMojoTestable(remoteExecutionManagerMock, remoteAutomationPackageClientMock);
 		configureMojo(mojo, true);
 		mojo.execute();
-
-		assertAutomationPackageClientMockCalls(remoteAutomationPackageClientMock);
-	}
-
-	private static List<String> getExecuteAutomationPackageResult(List<Execution> executions) {
-		return executions.stream().map(e -> e.getId().toString()).collect(Collectors.toList());
-	}
-
-	@Test
```

Comment:
@iegorov777 I've the feeling we now only have one happy path tests and removed all other negatives test (import error, execution failed..., is that expected